assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTile/AudienceTilePagesMetric.js

```diff
@@ -77,6 +83,7 @@ export default function AudienceTilePagesMetric( {
 
 	const redirectURL = addQueryArgs( global.location.href, {
 		notification: 'audience_segmentation',
+		widgetArea: AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION,
```

Comment:
This fixes a bug I noticed where the custom dimension creation isn't auto-continued when returning to Site Kit from the OAuth flow if the Audiences Widget Area isn't currently in view. The fix is to scroll to the widget area upon returning to the dashboard, as we do for the error case (see `errorRedirectURL` below).